consensus/istanbul/validator/default.go

```diff
@@ -306,11 +308,10 @@ func (valSet *defaultSet) CalcProposer(lastProposer common.Address, round uint64
 	defer valSet.validatorMu.RUnlock()
 
 	if len(valSet.validators) == 0 {
-		valSet.proposer = nil
```

Comment:
Don't we need to set nil like valSet.proposer.Store(nil)?

Reply:
atomic.value can not store nil value

Reply:
got it.